ci/homebrew-tap/ci-deploy.sh

```diff
@@ -21,6 +21,7 @@ sha256=$(
   curl -s https://$pypi_host/pypi/git-machete/$version/json \
   | jq --raw-output '.urls | map(select(.packagetype == "sdist")) | .[0].digests.sha256')
 sed -i "s/git-machete-.*\.tar\.gz/git-machete-$version.tar.gz/" git-machete.rb
+sed -i "s/pypi.io/$pypi_host/" git-machete.rb
```

Comment:
Nit: I'd escape `.` since it's not treated as literal period here, but rather as a regex single-char placeholder

Reply:
Hmmm why double nesting? one level being `bash -c`, the other being `$(...)`

Reply:
Resolved? 🤔